### PR TITLE
fix(theme): replace angle brackets in inline markdown

### PR DESCRIPTION
**Pull request by 10Derozan**
## Checklist

- [x] I have added changeset via `pnpm run change`.
- [ ] I have updated the documentation.
- [ ] I have added tests to cover my changes.

### CODE REVIEW

File: packages/theme-default/src/logic/utils.ts [hunk 1]
```diff
@@ -57,6 +57,7 @@ export function renderHtmlOrText(str?: string | number | null) {
  */
 export function renderInlineMarkdown(text: string) {
   const htmlText = text
+    .replace(/</g, '&lt;')
```
Comment (Huxpro) [inline packages/theme-default/src/logic/utils.ts:60]: what about `>` and `&gt;`?

**Review by Huxpro — COMMENTED** [review]
Have you tested on local that this works for the internal website?

File: packages/theme-default/src/logic/utils.ts (hunk 1 above)
Comment (10Derozan) [reply packages/theme-default/src/logic/utils.ts:60]: fine

**Comment by 10Derozan** [comment]
> Hope this works! Have you tested locally that this works for the internal website?

yeah, I have tested yet.

File: packages/theme-default/src/logic/utils.ts (hunk 1 above)
Comment (10Derozan) [reply packages/theme-default/src/logic/utils.ts:60]: only replace `<`, it works.

**Review by sanyuan0704 — APPROVED** [review]

**Pull request merged**